core/src/OpenMPTarget/Kokkos_OpenMPTarget_ParallelFor_Team.hpp

```diff
@@ -136,12 +136,12 @@ class ParallelFor<FunctorType, Kokkos::TeamPolicy<Properties...>,
     // If the league size is <=0, do not launch the kernel.
     if (max_active_teams <= 0) return;
 
-// Performing our own scheduling of teams to avoid separation of code between
-// teams-distribute and parallel. Gave a 2x performance boost in test cases with
-// the clang compiler. atomic_compare_exchange can be avoided since the standard
-// guarantees that the number of teams specified in the `num_teams` clause is
-// always less than or equal to the maximum concurrently running teams.
-#if !defined(KOKKOS_IMPL_OPENMPTARGET_HIERARCHICAL_INTEL_GPU)
+    // Performing our own scheduling of teams to avoid separation of code
+    // between teams-distribute and parallel. Gave a 2x performance boost in
+    // test cases with the clang compiler. atomic_compare_exchange can be
+    // avoided since the standard guarantees that the number of teams specified
+    // in the `num_teams` clause is always less than or equal to the maximum
+    // concurrently running teams.
```

Comment:
Please undo the changes to the comment formatting to avoid unnecessary noise in the history
```suggestion

// Performing our own scheduling of teams to avoid separation of code between
// teams-distribute and parallel. Gave a 2x performance boost in test cases with
// the clang compiler. atomic_compare_exchange can be avoided since the standard
// guarantees that the number of teams specified in the `num_teams` clause is
// always less than or equal to the maximum concurrently running teams.
```

Reply:
Ugggg.  That change was actually necessary for the format to pass...